flipdisc/framework/app.py

```diff
@@ -24,7 +24,7 @@ class Application(object):
     def __init__(
         self,
         name: str,
-        config: str,
+        config: Dict[Any, Any],
```

Comment:
It's a bit uglier, but I think it's better to make this a Union is better than having to `type: ignore` a call that passes in a config file name as a string.

```suggestion
        config: Union[Dict[Any, Any], str],
```

Reply:
ah that's cool that you can do Union, I tried doing `Dict[Any, Any] | str` and it said that character operator was introduced in a later python version

Reply:
actually, this threw errors everywhere when accessing `config` later in the file :/

Reply:
![Screen Shot 2022-06-28 at 2 51 35 PM](https://user-images.githubusercontent.com/1285301/176260032-4d8cfe6c-5590-49d1-a298-1d63f2449de1.png)

Reply:
Ah yeah, the `self.config = ...` line probably needs to be modified so it doesn't stay a Union type. This feels a little convoluted; it may be better to have a separate function that takes either a string or a Dict and always returns a Dict to keep things cleaner.